infra/container-registry/application-container-registry.bicep

```diff
@@ -0,0 +1,66 @@
+@description('Required. The parameter object for the virtual network. The object must contain the name,skuName,resourceGroup and subnetPrivateEndpoints values.')
+param vnet object
+
+@description('Required. The parameter object for container registry. The object must contain the name, enableSoftDelete, enablePurgeProtection and softDeleteRetentionInDays values.')
+param containerRegistry object
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Required. Environment name.')
+param environment string
+
+@description('Optional. Enable a single data endpoint per region for serving data. Not relevant in case of disabled public access. Note, requires the \'acrSku\' to be \'Premium\'.')
+param dataEndpointEnabled bool = false
+
+@description('Optional. Resource tags.')
+param tags object = {
+  Description: 'CDO Container Registry'
+}
+
+@description('Required. Date in the format yyyyMMdd-HHmmss.')
+param deploymentDate string = utcNow('yyyyMMdd-HHmmss')
```

Comment:
Can you please change the description for deploymentDate and createdDate to Optional?

Reply:
Ok will look at it @ppotturi

Reply:
Done now @ppotturi